src/map.cpp

```diff
@@ -602,59 +602,49 @@ float map::vehicle_vehicle_collision( vehicle &veh, vehicle &veh2,
     // Vertical collisions will be simpler for a while (1D)
     if( !vertical ) {
         // For reference, a cargo truck weighs ~25300, a bicycle 690,
-        //  and 38mph is 3800 'velocity'
-        rl_vec2d velo_veh1 = veh.velo_vec();
-        rl_vec2d velo_veh2 = veh2.velo_vec();
+        //  Velocities are in 1/100 mph
+        // Masses are in kilograms
+        rl_vec2d velocity_veh1 = veh.velo_vec();
+        rl_vec2d velocity_veh2 = veh2.velo_vec();
         const float m1 = to_kilogram( veh.total_mass() );
         const float m2 = to_kilogram( veh2.total_mass() );
         //Energy of vehicle1 and vehicle2 before collision
-        float E = 0.5 * m1 * velo_veh1.magnitude() * velo_veh1.magnitude() +
-                  0.5 * m2 * velo_veh2.magnitude() * velo_veh2.magnitude();
-
-        // Collision_axis
```

Comment:
What happened to all this code?  Why did you remove it?

Reply:
It didn't go anywhere. Github just marked it removed due to small name change.